tasks/21_check-letters/check-letters.js

```diff
@@ -0,0 +1,13 @@
+function checkLetters(input) {
+  const arr = input;
+  arr[0] = arr[0].toUpperCase();
```

Comment:
What was changed?

Reply:
My mistake, already researched how arrays work and fixed to concat()